nucleus/pom.xml

```diff
@@ -257,12 +257,12 @@
         
         <!-- Microprofile -->
         <microprofile-config.version>1.3</microprofile-config.version>
-        <microprofile-opentracing.version>1.1</microprofile-opentracing.version>
+        <microprofile-opentracing.version>1.3</microprofile-opentracing.version>
```

Comment:
This is actually 1.3.1 compliant as well, is there anything stopping us updating this for the sake of clarity?

Reply:
It only got released 2 days ago, I didn't know it was out :P